src/featureFlags.js

```diff
@@ -6,25 +6,25 @@ import {
   UNLEASH_URL,
   UNLEASH_CLIENT_KEY,
   UNLEASH_POLLING_INTERVAL,
+  STAGE,
 } from './constants';
 
 export const Events = {
   WALLET_SERVICE_ENABLED: 'wallet-service-enabled',
 };
 
 export class FeatureFlags extends events.EventEmitter {
-  constructor(userId, network) {
+  constructor(userId) {
     super();
 
     this.userId = userId;
-    this.network = network;
-    this.walletServiceFlag = `wallet-service-mobile-${Platform.OS}-${this.network}.rollout`;
+    this.walletServiceFlag = 'wallet-service-mobile.rollout';
```

Comment:
:speech_balloon: Should `'wallet-service-mobile.rollout'` be a constant?

Reply:
Yes we should, done in ec92ae2975e97d578f63b50f3f3dbe4f0198c3ea